src/debugger/direct/reactNativeDirectDebugEntryPoint.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as fs from "fs";
+import * as path from "path";
+import { Telemetry } from "../../common/telemetry";
+import { TelemetryHelper } from "../../common/telemetryHelper";
+import { EntryPointHandler, ProcessType } from "../../common/entryPointHandler";
+import { ErrorHelper } from "../../common/error/errorHelper";
+import { InternalErrorCode } from "../../common/error/internalErrorCode";
+import { NullTelemetryReporter, ReassignableTelemetryReporter } from "../../common/telemetryReporters";
+import { DirectDebugAdapter } from "./directDebugAdapter";
+import { makeDirectSession } from "./directSessionWrapper";
+import { ChromeDebugSession } from "vscode-chrome-debug-core";
+import { DebugSession, OutputEvent, TerminatedEvent } from "vscode-debugadapter";
+import * as nls from "vscode-nls";
+const localize = nls.loadMessageBundle();
+
+const projectRoot = path.join(__dirname, "..", "..", "..");
+const version = JSON.parse(fs.readFileSync(path.join(projectRoot, "package.json"), "utf-8")).version;
```

Comment:
This line and [that](https://github.com/microsoft/vscode-react-native/pull/1088/files?short_path=c691ac6#diff-974a8d3442d4db53597de589ba7fab41R167) one does the same thing I guess. Please, make sure that we have already somewhere in our codebase a method that implements this and if we haven't let's add one otherwise let's reuse it.